### ISSUE TITLE
Update validation scripts for MIMIC-IV-ED v2.2

### ISSUE THREAD

**Issue by ZhipengHe** [MIT-LCP/mimic-code#1475]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

[MIMIC-IV-ED v2.2 ](https://physionet.org/content/mimic-iv-ed/2.2/) removed a subset of subject_id which will be retained internally as a test set. Future data releases will exclude these patients. It removed 22,625 `stay_id` from all tables.

The current validation scripts for both mysql and postgres are crafted for MIMIC-IV-ED v2.0. 
mysql: https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv-ed/buildmimic/mysql/validate.sql
postgres: https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv-ed/buildmimic/postgres/validate.sql

If I run the v, they do not work for MIMIC-IV-ED v2.2. 
```
psql -U postgres -d mimiciv -f .\validate.sql
```

```
Validation results:
    tbl    | expected_count | observed_count | row_count_check
-----------+----------------+----------------+-----------------
 diagnosis |         946692 |         899050 | FAILED
 edstays   |         447712 |         425087 | FAILED
 medrecon  |        3143791 |        2987342 | FAILED
 pyxis     |        1670590 |        1586053 | FAILED
 triage    |         447712 |         425087 | FAILED
 vitalsign |        1646976 |        1564610 | FAILED
(6 rows)
```

Please update count numbers in `validate.sql` or create a new `validate.sql` for new version.

**Comment by ZhipengHe** [comment]
I have submitted a pull request #1476 for updating validation scripts for MIMIC-IV-ED.

**Comment by alistairewj** [comment]
Closed by #1476 - thanks!
